config/locales/en/brexit_checker/results.yml

```diff
@@ -12,11 +12,12 @@ en:
       description_no_answers: |
         <p class="govuk-body-l">To change your answers, follow the link below.</p>
         <p class="govuk-body-l">This is a safe and secure service. We do not store any of your personal information.</p>
-      email_sign_up_title: What you need to do may change
+      email_sign_up_title: Stay up to date
+      email_sign_up_description: Sign up to get emails about Brexit changes that may affect you.
       email_sign_up_link: |
-        Subscribe to email updates about changes that may affect you and get a link to your results
+        Get email updates
       email_sign_up_link_no_actions: |
```

Comment:
We could remove this and [the corresponding helper](https://github.com/alphagov/finder-frontend/blob/27a74325d47f17a95f77126ab5817ee21a713201/app/helpers/brexit_checker_helper.rb#L51-L57), since the text is now invariant between the two scenarios. I think it would be confusing to leave it in, if someone came across this in future.

Reply:
Yep, I think this is OK to remove. It would be simple enough to re-add if we needed to 👍

Reply:
I've done that in a new commit: https://github.com/alphagov/finder-frontend/pull/2354/commits/845889464103e67f9234c44ff985943be35786d0.